src/main/resources/wayf_widget_template.js

```diff
@@ -62,18 +50,6 @@ function readLocalId() {
     return readCookieValue(LOCAL_ID_COOKIE_NAME);
 }
 
-function setLocalId(localIdValue) {
```

Comment:
@balaf do you think a client would find this method useful? It would allow the widget to control the get/set of the local ID rather than relying on client to know where to set it